### PR TITLE
Fixed js paths for webviewer

### PR DESCRIPTION
**Pull request by nymitr**

### CODE REVIEW

File: data/interfaces/default/header.html [hunk 1]
```diff
@@ -8,8 +8,8 @@
   <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
   <link rel="stylesheet" type="text/css" href="interfaces/default/css/materialize.min.css">
```
Comment (evilhero) [inline data/interfaces/default/header.html:9]: This should point to the theme dir and not just the default one if we're correcting stuff (```href = interfaces/${interface}/css/materialize.min.css```)

**Review by evilhero — CHANGES_REQUESTED** [review]
PR is done against master branch, not python3-dev. Please resubmit against the python3-dev branch as we don't merge PR's directly into master. Cheers

**Comment by nymitr** [comment]
Sorry, I wasn't clear which branch should have been used.
Resubmitted.

File: data/interfaces/default/header.html (hunk 1 above)
Comment (nymitr) [reply data/interfaces/default/header.html]: Done.

**Review by evilhero — APPROVED** [review]

**Comment by evilhero** [comment]
Good to go in development now. Thanks for the PR and the additional changes ;)

**Pull request merged**